pages/about/index.tsx

```diff
@@ -32,6 +33,54 @@ interface TeamSection {
   members: string;
 }
 
+type ArticleType = {
+  title: string;
+  url: string;
+  sourceName: string;
+  lang: string | null; // TODO Language type
+  publishedDate: string;
+};
+
+type ArticlesListProps = {
+  articles: ArticleType[];
+};
+
+type ArticleProps = {
+  article: ArticleType;
+};
+
+const Article = ({ article }: ArticleProps): JSX.Element => {
+  const handleArticleClick = (e: React.MouseEvent<HTMLElement, MouseEvent>, url: string) => {
+    e.stopPropagation();
+    url && window.open(url);
+  };
+  return (
+    <div
+      className="w-full lg:max-w-full lg:flex cursor-pointer group px-2 py-1 my-1 rounded shadow border"
+      onClick={(e) => handleArticleClick(e, article.url)}
```

Comment:
Instead of a `div` with an event handler, you want to just use a regular `<a href={article.url} target="_blank" noreferrer>` element :)

Reply:
ok, i will change it to simple link